internal/bufferedread/buffered_reader.go

```diff
@@ -103,6 +103,61 @@ func NewBufferedReader(object *gcs.MinObject, bucket gcs.Bucket, config *Buffere
 	return reader, nil
 }
 
+func (p *BufferedReader) prefetch() error {
+	// Do not schedule more blocks if the prefetch queue has reached its maximum configured size.
+	availableSlots := p.config.MaxPrefetchBlockCnt - int64(p.blockQueue.Len())
+	if availableSlots <= 0 {
+		return nil
+	}
+	blockCountToPrefetch := min(p.numPrefetchBlocks, availableSlots)
+	if blockCountToPrefetch <= 0 {
+		return nil
+	}
+
+	logger.Debugf("Prefetching %d blocks", blockCountToPrefetch)
```

Comment:
```suggestion
	logger.Tracef("Prefetching %d blocks", blockCountToPrefetch)
```

Reply:
Doesn't seem very important log. Any way we will go over all the logs and see holistically if we can minimize the less important one.